Backend/app/dashboard/CommonAPIs/src/Dashboard/ProviderPlatform/Driver.hs

```diff
@@ -662,6 +662,41 @@ instance HideSecrets AddVehicleReq where
   hideSecrets = identity
 
 ---------------------------------------------------------
+
+-- add vehicle for fleet ------------------------------------------
+
+type AddVehicleForFleetAPI =
+  Capture "mobileNo" Text
+    :> "addVehicle"
+    :> "fleet"
+    :> ReqBody '[JSON] AddVehicleReq
+    :> Post '[JSON] APISuccess
+
+---------------------------------------------------------
+
+-- get vehicle for fleet  ------------------------------------------
+
+type GetAllVehicleForFleetAPI =
+  "getAllVehicle"
+    :> "fleet"
+    :> Get '[JSON] ListVehicleRes
+
+newtype ListVehicleRes = ListVehicleRes
+  {vehicles :: [VehicleAPIEntity]}
+  deriving (Generic, ToJSON, ToSchema, FromJSON)
+
+data VehicleAPIEntity = VehicleAPIEntity
+  { driverId :: Text,
```

Comment:
@AshishSaini25 is there a seperate api to get driver status, earning , trips completed